packages/upgrade/src/codemods/__tests__/__fixtures__/transform-clerk-react-v6.fixtures.js

```diff
@@ -0,0 +1,77 @@
+export const fixtures = [
```

Comment:
I think the answer is no here, but do we care about codemoding CJS code with require? 

```
const { useSignIn } = require('@clerk/clerk-react');
```